src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -6325,12 +6325,15 @@ namespace HVACVariableRefrigerantFlow {
 					SPTempLo = ZoneThermostatSetPointLo( ThisZoneNum );
 					{ auto const SELECT_CASE_var( TempControlType( ThisZoneNum ) );
 					if ( SELECT_CASE_var == 0 ) { // Uncontrolled
+					// MaxDeltaT denotes cooling, MinDeltaT denotes heating
 					} else if ( SELECT_CASE_var == SingleHeatingSetPoint ) {
-						ZoneDeltaT = max( 0.0, SPTempLo - ZT( ThisZoneNum ) );
-						MaxDeltaT( VRFCond ) = max( MaxDeltaT( VRFCond ), ZoneDeltaT );
-					} else if ( SELECT_CASE_var == SingleCoolingSetPoint ) {
-						ZoneDeltaT = min( 0.0, SPTempHi - ZT( ThisZoneNum ) );
+						// if heating load, ZoneDeltaT will be negative
+						ZoneDeltaT = min( 0.0, ZT( ThisZoneNum ) - SPTempLo );
 						MinDeltaT( VRFCond ) = min( MinDeltaT( VRFCond ), ZoneDeltaT );
+					} else if ( SELECT_CASE_var == SingleCoolingSetPoint ) {
+						// if cooling load, ZoneDeltaT will be positive
+						ZoneDeltaT = max( 0.0, ZT( ThisZoneNum ) - SPTempHi );
+						MaxDeltaT( VRFCond ) = max( MaxDeltaT( VRFCond ), ZoneDeltaT );
```

Comment:
These were backwards. If you look below at line 6478 and 6481, you see that MaxDeltaT > 0 represent cooling and MinDeltaT < 0 represent heating. I fixed the math and used the correct variables.